cpp/include/cugraph_c/layout_algorithms.h

```diff
@@ -47,21 +47,28 @@ typedef struct {
  * @param [in]   graph           Pointer to graph.  NOTE: Graph might be modified if the storage
  *                               needs to be transposed
  * @param [in]   max_iter        Maximum number of iterations. Initial vertex positioning
+ * @param [in]   start_vertices  Optionally send in an initial vertex positioning.
+                                 This defines the vertices for x_start and y_start.
  * @param [in]   x_start         Optionally send in an initial vertex positioning (x-axis).
  *                               Set to NULL if no value is passed. If NULL, the initial x-axis
  *                               are selected randomly.
- *                               x_start is sorted by vertex id
  * @param [in]   y_start         Optionally send in an initial vertex positioning (y-axis).
  *                               Set to NULL if no value is passed. If NULL, the initial y-axis
  *                               are selected randomly.
- *                               y_start is sorted by vertex id
  * @param [in]   outbound_attraction_distribution
  *                               Distributes attraction along outbound edges
  *                               Hubs attract less and thus are pushed to the borders.
  * @param [in]   lin_log_mode    Switch Force Atlas model from lin-lin to lin-log.
  *                               Makes clusters more tight.
  * @param [in]   prevent_overlapping
  *                               Prevent nodes to overlap.
+ * @param [in]   vertex_radius_vertices
```

Comment:
Do we want to make these separate from `start_vertices`?

Is there a use case where you specify x/y values and don't specify a vertex radius value, or vice verse?  Same question for mobility values.

I guess I'm less concerned with things that are O(number_of_vertices), but it does cost us GPU memory, and since they're managed by python the memory will be used for the duration of the call.